### ISSUE TITLE
Infinite scrolling only works if there's a scroll bar

### ISSUE THREAD

**Issue by crazedVic** [redbastie/tailwire#8]
Default page size is set to 15, if the table rows are quite compact, and vertical real estate is great enough, then index will render the first 15 records on the page, there will be no scroll bar, and that's it, you're kinda done.

This makes sense of course, however since we can't be sure of the vertical height of any client screen, nor the font size they are using, it could well happen a client has the room to render 15 rows without causing scroll bars to appear.

Easiest solution might be to have a load more button at the bottom that renders as long as getInfiniteScrollProperty() returns true.

```
@if($this->infiniteScroll)
            <div class="flex justify-center mx-auto w-full my-2 py-8">
                <a class="bg-black text-white rounded-md shadow-md px-2 py-1" wire:click.prevent="infiniteScroll" href="">Load More</a>
            </div>
@endif
```

**Comment by redbastie** [comment]
Yeah that is a good idea. Have a button as a fallback. I'll work on it.

**Comment by redbastie** [comment]
Just released `2.1.11` which has a new component property `perPageIncrement`.

So right now, it defaults to `15`. If you want to increase it, just make the values larger for the initial `perPage`, and the `perPageIncrement`:

```
class MyComponent extends Component
{
    public $perPage = 15;
    public $perPageIncrement = 15;
```

For example, change `15` to `30` for both if you're finding your page isn't long enough to have a scrollbar.
